lib/chef/azure/commands/enable.rb

```diff
@@ -92,7 +114,24 @@ def install_chef_service
   end
 
   def enable_chef_service
-    @exit_code, error_message = ChefService.new.enable(@chef_extension_root, bootstrap_directory, @azure_plugin_log_location)
+    chef_service_interval = load_chef_service_interval
+
+    if chef_service_interval.empty?
+      @exit_code, error_message = ChefService.new.enable(@chef_extension_root, bootstrap_directory, @azure_plugin_log_location)

```

Comment:
should we treat empty the same as 0? Perhaps it was running before and instead of setting to 0, it was removed. Note this is a question as I'm not extremely familiar with the use case.